.travis.yml

```diff
@@ -1,22 +1,13 @@
 sudo: false
 language: node_js
 node_js:
-- '0.12'
+- '5.9'
```

Comment:
It depends on what happens between now and October I believe, as the LTS schedule says the next LTS version starts on Oct. 1.